.github/workflows/release.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       id: semver-tag-dry 
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        WITH_V: false
+        WITH_V: true
```

Comment:
While this looks good, though since this is the change to the version tagging format, I'd suggest to postpone this item to some time later as we need to consider how much this is beneficial for us to have `tfswitch` as a Go module.
@warrensbox What do you think?

Reply:
I think the reason people want this updated is so `go get` works as expected. I don't really see a disbenefit, as long as changing the format doesn't breaking anything downstream

Reply:
Unfortunately we can't guarantee none of the consumers/downstream rely upon versioning without `v` as prefix.
It's also off that they require(?) `v` whilst it is not a part of valid semver 😕

Reply:
It's a bit odd, but if you have a look into the go.mod you'll see that all the modules have versions with a "v" Prefix
So it seems to be the thing GO is versioning 🤷

Reply:
> all the modules have versions with a "v" Prefix

`tfswitch` is an app firstly as opposite to Go module 😺 
Let's see what @warrensbox thinks of this.

Reply:
> Unfortunately we can't guarantee none of the consumers/downstream rely upon versioning without v as prefix.

Sure, but this isn't breaking previous releases. It's just a breaking change going forwards if you're relying on a specific format to the version number. 
I wouldn't be very concerned about changing that convention. 

> is an app firstly as opposite to Go module 

Agreed. Not too fussed either way - maybe if someone heavily in the Go community chimes in on whether we should/should not be following the module convention as an app that'd help.

Reply:
> It's just a breaking change going forwards if you're relying on a specific format to the version number.

Yep, this is what I mean.

> I wouldn't be very concerned about changing that convention.

If we announce that and provide some time, then we probs can break what's been in place from the very beginning...
Either way I'd leave this for @warrensbox to consider and decide.

Reply:
I agree with @yermulnik; we should provide a heads-up in the README.md about this. That way, people who have been using it without the 'v' can adjust their workflow accordingly.

I also agree that tfswitch if more of a tool rather than a module.

Not super urgent, but we can announce that we are switching on the README, and set a target version that would include "v"

Reply:
I still need to update the README on this